spec/openapi.yaml

```diff
@@ -115,6 +115,14 @@ paths:
           schema:
             type: boolean
             default: False
+        - in: header
+          name: Overwrite
+          description: Whether to overwrite already existing files
+          schema:
+            type: string
+            description: ?0=False, ?1=True, according RFC8941/3.3.6
+            enum: ["?0", "?1"]
+            default: "?0"
```

Comment:
Is there a good reason why we have several `True/False` headers and one that's a bit weird `?0` and `?1` (not even pure `0`/`1`) :-O

Reply:
Yes that RFC 8941 and it's section 3.3.6.
We try to find some standard solution, and that is described in RFC.